PED/SetIkTarget.md

```diff
@@ -5,18 +5,68 @@ ns: PED
 
 ```c
 // 0xC32779C16FCEECD9 0x6FE5218C
-void SET_IK_TARGET(Ped ped, int ikIndex, Entity entityLookAt, int boneLookAt, float offsetX, float offsetY, float offsetZ, Any p7, int blendInDuration, int blendOutDuration);
+void SET_IK_TARGET(Ped ped, int ikIndex, Entity entityLookAt, int boneLookAt, float offsetX, float offsetY, float offsetZ, int ikTargetFlags, int blendInDuration, int blendOutDuration);
 ```
 
+Sets the IK target for a given IK part belonging to the ped. The IK target will only be valid for one update, so it needs to be set for as long as it is needed (to avoid IK targets not being cleared and getting stuck enabled).
+
 ## Parameters
-* **ped**: 
-* **ikIndex**: 
-* **entityLookAt**: 
-* **boneLookAt**: 
-* **offsetX**: 
-* **offsetY**: 
-* **offsetZ**: 
-* **p7**: 
+* **ped**: The ped handle.
+* **ikIndex**: See ``IK_PART``.
+* **entityLookAt**: Set to 0 for no entity, offsets will be world coordinates instead.
+* **boneLookAt**: Set to -1 for no target bone.
+* **offsetX**: X-axis offset calculated from the position of ``entityLookAt``.
+* **offsetY**: Y-axis offset calculated from the position of ``entityLookAt``.
+* **offsetZ**: Z-axis offset calculated from the position of ``entityLookAt``.
+* **ikTargetFlags**: See ``IK_TARGET_FLAGS``.
```

Comment:
Most of the docs use only 1 backticks `` ` ``, it's not include in signature method but for better standart.